test_upper_case_names (__main__.TestExercice) ... ERROR

======================================================================
ERROR: test_upper_case_names (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 17, in test_upper_case_names
    output = list(map(exercice.majuscule, self.words))
  File "/home/runner/work/c02-ch2-majuscule-matben001/c02-ch2-majuscule-matben001/exercice.py", line 10, in majuscule
    resultat += lettre
TypeError: must be str, not int

----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
